client/package.json

```
@@ -26,16 +27,15 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint 'src/**/*.{js,jsx}'",
-    "format": "prettier --write \"src/**/*.{js,jsx,json,css}\"",
-    "format:check": "prettier --check 'src/**/*.{js,jsx,json,css}'",
-    "prepare": "cd .. && husky install client/.husky"
+    "lint": "eslint .",
+    "lint:fix": "eslint --cache --fix",
```

Could we potentially swap the two names, because `yarn lint` is easier to type? 

Husky doesn't seem to fix eslint warnings so we might be typing this command a lot. 

```suggestion
    "lint": "eslint --cache --fix",
    "lint:nofix": "eslint .",
```

---

is there any other warnings that eslint did not fix (because it should fix them)? the jsxquote wasnt fixed because prettier overwrite the fix. normally, `yarn lint` is used just to check. ideally, you dont have to manually run those checks, because that is what pre-commit and GitHub action is for